src/main/scala/com/chaordicsystems/cactus/Operation.scala

```diff
@@ -0,0 +1,26 @@
+package com.chaordicsystems.cactus
+
+  object Operation extends Enumeration {
```

Comment:
I prefer this way because of [this](https://stackoverflow.com/a/1321831). I do think is more natural since we're parsing json-like strings.